server/repository/src/db_diesel/store_row.rs

```diff
@@ -91,9 +89,12 @@ impl<'a> StoreRowRepository<'a> {
         Ok(result)
     }
 
-    pub fn find_one_by_name_id(&self, name_id: &str) -> Result<Option<StoreRow>, RepositoryError> {
+    pub fn find_one_by_name_id(
+        &self,
+        name_link_id: &str,
```

Comment:
Maybe rename the method as well. Not sure if we want to do this though, in general we want to lookup by name_id not by name_link_id... (see other comment)